### PR TITLE
feat: adds token balances endpoint

### PR DESCRIPTION
**Pull request by nicholaswma**
@7i7o **This only returns AO tokens and balances - do we want to return balance for AR?**

TODO: we'll need to update types and docs

### CODE REVIEW

**Review by 7i7o — APPROVED** [review]
tACK 👍

File: src/api/modules/user_tokens/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { PermissionType } from "~applications/permissions";
+import type { ModuleProperties } from "~api/module";
+
+const permissions: PermissionType[] = ["ACCESS_BALANCES"];
```
Comment (pawanpaudel93) [inline src/api/modules/user_tokens/index.ts]: Should this permission still be called `ACCESS_BALANCES` when the api is renamed ? What about `ACCESS_TOKENS` ?

Comment (pawanpaudel93) [reply src/api/modules/user_tokens/index.ts]: @7i7o Thoughts?

Comment (7i7o) [reply src/api/modules/user_tokens/index.ts]: I agree with @pawanpaudel93 that `ACCESS_TOKENS` is a better name for the added permission

**Review by pawanpaudel93 — APPROVED** [review]
tACK 🚀

Just a suggestion for the permission text: 
> Allows access to view tokens and their balances in your wallet.

**Review by 7i7o — APPROVED** [review]
tACK 👍

**Pull request merged**